Content.Shared/Borgs/LawMessages.cs

```diff
@@ -0,0 +1,35 @@
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Borgs
+{
+    [NetSerializable, Serializable]
+    public enum LawsUiKey : byte
+    {
+        Key,
+    }
+
+    [Serializable, NetSerializable]
+    public sealed class LawsUpdateState : BoundUserInterfaceState
+    {
+        public HashSet<string> Laws;
```

Comment:
You're eating order information here and hoping it works. Ideally this is fixed as part of the aforementioned changes to how you store laws.